app/i18n/nl/conf.php

```diff
@@ -39,6 +39,7 @@
 		),
 		'show_nav_buttons' => 'Toon navigatieknoppen',
 		'theme' => 'Thema',
+		'theme_not_available' => 'The "%s" theme is not available anymore. Please choose another theme.', // TODO - Translation
```

Comment:
Oh right, might as well add the Dutch translation as part of the PR.
```suggestion
		'theme_not_available' => 'Het "%s" thema is niet meer beschikbaar. Kies een ander thema.',Translation
```

Reply:
thanks. done.